src/ORM/Connect/DBSchemaManager.php

```diff
@@ -545,40 +544,6 @@ protected function determineIndexType($spec)
     }
 
     /**
-     * Converts an array or string index spec into a universally useful array
-     *
-     * @see convertIndexSpec() for approximate inverse
-     * @param string $name Index name
-     * @param string|array $spec
-     * @return array The resulting spec array with the required fields name, type, and value
-     */
-    protected function parseIndexSpec($name, $spec)
```

Comment:
Per my comment on `index` being default, maybe keep this method, but rename it to `normaliseIndexSpec` and remove the support for non-array format?